pkg/kubectl/resource/visitor.go

```diff
@@ -97,6 +98,37 @@ func (i *Info) Get() error {
 	return nil
 }
 
+// Refresh updates the object with another object. If ignoreError is set
+// the object will be updated directly
```

Comment:
I think this should say "If ignoreError is set, the object will be updated even if metadata cannot be read from the object". The object will still be updated directly if ignoreError is False and metadata can be read.